common/changes/@rushstack/node-core-library/user-ianc-move-terminal_2021-06-14-00-18.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/node-core-library",
+      "comment": "(BREAKING CHANGE) Move the Terminal APIs out of node-core-library to the @rushstack/terminal package.",
+      "type": "minor"
```

Comment:
Should this be major? We'll have to do a lot of rewriting code in many plugins and utilities.

Reply:
Yeah probably. Terminal is marked as `@beta`, but it's pretty heavily used at this point.